statsmodels/tsa/arima/model.py

```diff
@@ -44,19 +44,23 @@ class ARIMA(sarimax.SARIMAX):
 
     Parameters
     ----------
-    endog : array_like, optional
+    endog : array_like
         The observed time-series process :math:`y`.
     exog : array_like, optional
         Array of exogenous regressors.
-    order : tuple, optional
+    order : tuple, default: (0, 0, 0)
         The (p,d,q) order of the model for the autoregressive, differences, and
         moving average components. d is always an integer, while p and q may
-        either be integers or lists of integers.
-    seasonal_order : tuple, optional
+        either be integers or lists of integers specifying exactly which lag
+        orders are included. The order of differences is to achieve
+        stationnarity in the context of a sochastic trend or seasonality. If
+        your trend in deterministic, use the `trend` parameter as it will
```

Comment:
On the one hand, I feel like `d` is so often mistaken to model a deterministic trend that it has his place under the spotlight. Only a handful of people go to see examples while most of them (us) stop at the API definition. On the other hand, now that 'sochastic' is mentioned, it may raise enough questions in the user's mind to stop him from using `d`wrongfully.